OperatingSystem-Services/Platform-Windows/CA/sign-csr-file.md

```diff
@@ -0,0 +1,26 @@
+
+Now that you have the .csr file on your Windows machine, you can submit the request and generate a certificate. From the Windows Server machine, CA can be opened by typing "Certification Authority" and opening from the search bar, or intuitively through Server Manager by navigating to "AD CS" on the left side bar, right clicking the current server, and clicking "Certification Authority."
+
+![ss2](Images/search.png)
+
+From inside the Certification Authority service, the CA which was created should now be an entry on the left side bar. To generate a certificate using this CA, right click, hover over "All Tasks," and click "Submit new request..."
+
+![ss2](Images/submit.png)
+
+This should open a file explorer tab, in which you must navigate to your .csr file. One thing to note is you must change the file types in this file explorer tab to "All files" rather than the select few extensions. Find your .csr file, and select open.
```

Comment:
```suggestion
This should open a file explorer tab, in which you must navigate to your `.csr` file.  Find your `.csr ` file, and select open.

> [!NOTE]
> You must change the file types in this file explorer tab to "All files" rather than selecting a few extensions.
```